public/js/AdminCtrl.js

```diff
@@ -264,14 +264,16 @@ polr.controller('AdminCtrl', function($scope, $compile, $timeout) {
 
     // Delete user
     $scope.deleteUser = function($event, user_id) {
-        var el = $($event.target);
-
-        apiCall('admin/delete_user', {
-            'user_id': user_id,
-        }, function(new_status) {
-            toastr.success('User successfully deleted.', 'Success');
-            $scope.reloadUserTables();
-        });
+        if(confirm("Are you sure to delete this user?")){
```

Comment:
Can we do something like
```
var delete_confirm = window.confirm("...");
```
instead of nesting the `confirm` statement within the `if(` statement? 

Also, let's reword the query so that it is gramatically correct. What about `"Are you sure you would like to delete this user?"`?